com.eclipsesource.modelserver.emf/src/main/java/com/eclipsesource/modelserver/emf/common/SessionController.java

```diff
@@ -15,32 +15,42 @@
  *******************************************************************************/
 package com.eclipsesource.modelserver.emf.common;
 
-import static java.util.stream.Collectors.toSet;
+import com.eclipsesource.modelserver.emf.common.codecs.Encoder;
+import com.eclipsesource.modelserver.emf.common.codecs.EncodingException;
+import com.eclipsesource.modelserver.jsonschema.Json;
+import com.google.common.collect.Maps;
+import com.google.inject.Inject;
+import io.javalin.http.Context;
+import io.javalin.websocket.WsContext;
+import io.javalin.websocket.WsHandler;
+import org.apache.log4j.Logger;
+import org.eclipse.emf.ecore.EObject;
+import org.jetbrains.annotations.Nullable;
+import org.jetbrains.annotations.TestOnly;
 
 import java.util.Iterator;
 import java.util.Map;
 import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.stream.Stream;
 
-import org.eclipse.emf.ecore.EObject;
-import org.jetbrains.annotations.Nullable;
-import org.jetbrains.annotations.TestOnly;
-
-import com.eclipsesource.modelserver.jsonschema.Json;
-import com.google.common.collect.Maps;
-import com.google.inject.Inject;
-
-import io.javalin.websocket.WsContext;
-import io.javalin.websocket.WsHandler;
+import static java.util.stream.Collectors.toSet;
 
 public class SessionController extends WsHandler {
 
+	Logger LOG = Logger.getLogger(SessionController.class);
```

Comment:
Here `SessionController.class.getSimpleName()` could be used (as in other classes) to reduce the length of the LOG line.